.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@
 
 - [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
 - [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
-- [ ] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
+- [ ] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides]. (Note that not like the flutter/flutter repo, we do use dart formatter in the flutter/plugins repo.)
```

Comment:
```
[ ] I have formatted this PR with `dartfmt -w .`, read the [Flutter Style Guide] and followed the [C++, Objective-C, Java style guides].
```

Is there a plugins command that we can provide so users can quickly format all the source they might have touched in their PR? Something that guarantees that the `format` step will pass?

Reply:
does it make sense to say `dart run ./script/tool/lib/src/main.dart format` this those all of those.

Reply:
done